internal/rbd/rbd_util.go

```diff
@@ -564,6 +564,16 @@ func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 
 		return err
 	}
+	defer func() {
```

Comment:
Indention looks really weird in GitHub while reviewing this change... I wonder if there is some strange space/tab mismatch here.

Reply:
>Indention looks really weird in GitHub while reviewing this change... I wonder if there is some strange space/tab mismatch here.

Double checked, looks fine to me. Maybe because of nesting depth, it looks that way.